README.md

```diff
@@ -1,54 +1,28 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/hvb5pfagqejwvihs/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/versioning_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/Versioning_Toolkit/Versioning_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=128&branchName=master)
 
-A great place to start is reading our Wiki [here](https://github.com/BHoM/documentation/wiki) including pages like the [Structure of the BHoM](https://github.com/BHoM/documentation/wiki/Structure-of-the-BHoM) and [Using the BHoM](https://github.com/BHoM/documentation/wiki/Using-the-BHoM).
+# Versioning_Toolkit
 
-## Quick start ##
+Provide ability for serialised items (methods, types, BHoM objects) to automatically upgrade themselves in case their code definition has changed since they were serialised.
```

Comment:
Would it be better to say `upgrade themselves when their code definition has changed`, given we do the upgrade when code changes rather? Unless I misunderstood the logic steps for versioning, I thought we deserialised into the BHoM object, and if it couldn't be found (cause of code change) then we run it through versioning to find it and upgrade it.

It's a slight nuance of language I'll admit, so not pushing for a change, just a suggestion/discussion point, I'd be happy to accept the PR with this line as is :smile:

Reply:
the upgrade of the json happens when the object is deserialised, not when the code is modified. 
Re-reading this now, I would probably prefer 'if' instead of 'in case' but I otherwise think what I have written is correct.
Let me know if I missed your point. I'll leave this open for now.

Reply:
I am happy with the use of `if` instead of `in case`, that reads better in my mind, will mark this as resolved.